.github/workflows/image-publish.yml

```diff
@@ -81,3 +96,18 @@ jobs:
         run: |
           SHORT_SHA1=$(git rev-parse --short=7 HEAD)
           docker buildx build --platform linux/amd64 -f build/dockerfiles/dev.Dockerfile --push -t quay.io/che-incubator/che-code-dev:insiders -t quay.io/che-incubator/che-code-dev:next -t quay.io/che-incubator/che-code-dev:insiders-${SHORT_SHA1} .
```

Comment:
you've already done this line above on line 79.

Reply:
oops, I see this is for che-code-dev, not che-code. Does this need to be done BEFORE the che-code image?